Bottom: c10fb3055e8ab5734c387534621b986449fd6996
Top:    13bb73e6d5fffb24e2af12520bbe2fb35a12eef7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-27 15:45:24 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 525dd93..5b05e36 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -494,7 +494,7 @@ void ghid_logv (const char *fmt, va_list args);
 /* gui-pinout-window.c */
 void ghid_pinout_window_show (GHidPort * out, ElementTypePtr Element);
 
-/* gtkhid-gdk.c OR gtkhid-gl.c */
+/* gtkhid-gdk.c AND gtkhid-gl.c */
 hidGC ghid_make_gc (void);
 void ghid_destroy_gc (hidGC);
 void ghid_use_mask (int use_it);
